src/pages/User/components/PullRequests/hooks/useFetchPullRequests.js

```diff
@@ -34,19 +43,36 @@ export default function useFetchPullRequests(username) {
   useEffect(() => {
     if (username === '') return;
     dispatch({ type: ACTIONS.PULL_REQUESTS_LOADING });
-    fetchPullRequests(username)
-      .then((pullRequests) =>
-        dispatch({
-          type: ACTIONS.PULL_REQUESTS_FETCHED,
-          payload: { pullRequests },
-        })
-      )
-      .catch((error) =>
-        dispatch({
-          type: ACTIONS.PULL_REQUESTS_FETCH_ERROR,
-          payload: { error },
-        })
-      );
+    // fetchPullRequests(username)
+    //   .then((pullRequests) =>
+    //     dispatch({
+    //       type: ACTIONS.PULL_REQUESTS_FETCHED,
+    //       payload: { pullRequests },
+    //     })
+    //   )
+    //   .catch((error) =>
+    //     dispatch({
+    //       type: ACTIONS.PULL_REQUESTS_FETCH_ERROR,
+    //       payload: { error },
+    //     })
+    //   );
+
+    // code by Georgey
+    const response = await fetchPullRequests(username);
+    const data = dispatch({ 
+      type: ACTIONS.PULL_REQUESTS_FETCHED,
+      payload: {response},
+    })
+
+    const errors = dispatch({ 
+      type: ACTIONS.PULL_REQUESTS_FETCH_ERROR,
+      payload: {data}
+    })
+
+    if(errors) {
```

Comment:
Think we're missing a space in here, you should be able to run `npm run prettier-fix` to correct 👍